src/planning/plan/suggestion.ts

```diff
@@ -84,16 +84,16 @@ export class Suggestion {
   }
 
   getDescription(modality: string): string|{} {
-    assert(this.descriptionByModality[modality], `No description for modality '${modality}'`);
+    //assert(this.descriptionByModality[modality], `No description for modality '${modality}'`);
```

Comment:
is this commented out on purpose? is this `assert` firing? when?

Reply:
It's fired here: https://github.com/PolymerLabs/arcs/blob/dd5afda4bdfeb153ecef03cc8f7a38236aaaec57/src/planning/planner.ts#L186

Reply:
ah ok, i understand it now.
the original intention of this assert was to test the presence of the `modality` key in `this.descriptionByModality`, and i would still like to keep doing it. Now that the text description can be undefined, let's tweak the assert to be: `assert(this.descriptionByModality.hasOwnProperty(modality), ...)`

Reply:
OK, I'll add it back in another commit.

Reply:
So I tested it but got some exceptions when running sigh. I suspect that it's caused by constructing suggestions with fromLiteral (https://github.com/PolymerLabs/arcs/blob/bbfa719770abadfe8d3956a86fb590bdfb8d9e84/src/planning/plan/suggestion.ts#L191) as the descriptionByModality passed in could be empty.

Reply:
argh.. to avoid this i think we could `JSON.stringify` descriptionByModality` similarly to how we do with `versionByStore`, but i also think it is ok to leave it as is for now. 
thank you for handling this.

Reply:
Np, thanks for your help!